lib/trino-parquet/src/main/java/io/trino/parquet/reader/decoders/ApacheParquetValueDecoders.java

```diff
@@ -505,6 +507,82 @@ public void skip(int n)
         }
     }
 
+    public static final class DoubleApacheParquetValueDecoder
```

Comment:
TestDoubleValueDecoders should also use this class rather than duplicating it

Reply:
Done